test/clt-tests/tables-interaction/test-tables-interaction.rec

```diff
@@ -0,0 +1,91 @@
+––– output –––
+0
+––– input –––
+docker pull -q --platform linux/x86_64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker network create manticore_network > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker run -d --name manticore_remote --network manticore_network -v /tmp/local-index:/var/lib/manticore/ --platform linux/amd64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+if timeout 5 grep -qm1 'accepting connections' <(docker logs -f manticore_remote); then echo 'accepting connections'; else echo 'Timeout failed'; fi
+––– output –––
+accepting connections
+––– input –––
+docker run -d --name manticore_dist --network manticore_network -v /tmp/dist-index:/var/lib/manticore/ --platform linux/amd64 manticoresearch/manticore:6.2.12 > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+if timeout 5 grep -qm1 'accepting connections' <(docker logs -f manticore_dist); then echo 'accepting connections'; else echo 'Timeout failed'; fi
+––– output –––
+accepting connections
+––– input –––
+docker exec manticore_remote mysql -e "CREATE TABLE t (id BIGINT, title TEXT); INSERT INTO t (id, title) VALUES (1, 'Test title latest'), (2, 'Another test title latest');"
+––– output –––
+––– input –––
+docker exec manticore_remote mysql -e "SELECT * FROM t;"
+––– output –––
++------+---------------------------+
+| id   | title                     |
++------+---------------------------+
+|    1 | Test title latest         |
+|    2 | Another test title latest |
++------+---------------------------+
+––– input –––
+docker exec manticore_dist mysql -e "CREATE TABLE dist_table type='distributed' agent='manticore_remote:9312:t';"
+––– output –––
+––– input –––
+docker exec manticore_dist mysql -e "DESC dist_table"
+––– output –––
++-------------------------+----------+
+| Agent                   | Type     |
++-------------------------+----------+
+| manticore_remote:9312:t | remote_1 |
++-------------------------+----------+
+––– input –––
+docker exec manticore_dist mysql -e "select * from dist_table;"
+––– output –––
++------+---------------------------+
+| id   | title                     |
++------+---------------------------+
+|    1 | Test title latest         |
+|    2 | Another test title latest |
++------+---------------------------+
+––– input –––
+docker stop manticore_remote
+––– output –––
+manticore_remote
+––– input –––
+docker rm manticore_remote
+––– output –––
+manticore_remote
+––– input –––
+docker run -d --name manticore_remote --network manticore_network -v /tmp/local-index:/var/lib/manticore/ --platform linux/amd64 manticoresearch/manticore:latest > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+if timeout 5 grep -qm1 'accepting connections' <(docker logs -f manticore_remote); then echo 'accepting connections'; else echo 'Timeout failed'; fi
+––– output –––
+accepting connections
```

Comment:
What's going on here? Are you testing that it still works after it's restarted?

Reply:
We upgrade here agents version to latest